.github/workflows/publish_docker.yml

```diff
@@ -0,0 +1,36 @@
+name: Publish Docker image
+
+on:
+  push:
+    branches: [master]
```

Comment:
does this mean that the Formplayer will be continuously to production as soon as we merge into master ?

Reply:
not, it means that the docker image will be updated after each merge to master

Reply:
I see, purely for my own understanding it's only going to effect someone who runs FP on docker locally ? I am not sure where else does the FP docker image get used.

Reply:
currently the only use for the docker images is local development and the commcare-hq build